src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -129,28 +129,25 @@ Actors:
   - *Nop
   - *Nop
 
-# TODO: TIG-3050 Switch to CrudActor once projection argument is supported for find operation.
 - Name: RunQueries
-  Type: RunCommand
```

Comment:
Does the queries in this file actually require multiple batches? If there's no actual problem with this workload as originally implemented, I'd be inclined to just delete the TODO comment above and leave it as is. That way, we avoid any risk of a discontinuity in the performance data, but we've still done the work to improve the underlying genny infrastructure.